canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.scss

```diff
@@ -16,11 +16,11 @@
 
 $primary-tab-height: $spacing-08;
 
-.properties-primaryTabs {
-	.bx--tabs__nav-item {
-		margin-left: 1rem;
-	}
-}
+// .properties-primaryTabs {
```

Comment:
I'm assuming this was comment out because it didn't work. Can we add a TODO comment?

Reply:
This is an old code from 6 years ago which wasn't adding any CSS in carbon 10. I will remove this code. 
I will add a comment in `subtabs.scss` which has a known CSS issue in `properties-leftnav-container`.